src/Symfony/Component/Console/Helper/FormatterHelper.php

```diff
@@ -54,12 +54,12 @@ public function formatBlock($messages, $style, $large = false)
         foreach ($messages as $message) {
             $message = OutputFormatter::escape($message);
             $lines[] = sprintf($large ? '  %s  ' : ' %s ', $message);
-            $len = max($this->strlen($message) + ($large ? 4 : 2), $len);
+            $len = max(static::strlen($message) + ($large ? 4 : 2), $len);
```

Comment:
do we actually need to use late static binding here ?

Reply:
and is it related to this PR ? This looks like the only change in this file in this PR.

Reply:
It's not clear that' `strlen` s is static method, that not good to call static by $this

Reply:
I'm not objecting the fact of calling it statically. I'm objecting the fact of using late static binding for that (`static::` vs `self::`) and about changing files unrelated to what the PR is about (meaning that if you want to submit such change, it should be done on its own rather than bundling it in another PR)

Reply:
remove this back?

Reply:
Reverted back this changes